File: HackerRank/Rotation.py
def rotation(arr):
    n = len(arr)
    temp = arr[n-1]
    for i in range(n-1,0,-1):
        arr[i] = arr[i-1]
    arr[0] = temp
    return arr





for i in range(int(input())):
    n,k = map(int,input().split())
    arr = list(map(int,input().split()))

    for i in range(k):
        arr = rotation(arr)
    print(*arr)


    

File: rotation.cpp
#include <iostream>
using namespace std;


int main() {
	int t;
	for (int i = 0; i<t;i++)
	{
		int n,k;
		int arr[n];

		cin >> n >> k ;

		for (int i =0;i<n;i++)
		{
			cin >> arr[i];
		}


		for(int i = 0;i<k;i++)
		{
			int temp = arr[n-1];
			
			for (int j = n-1;j>0;j--)
			{
				arr[j] = arr[j-1];
			}
			arr[0] = temp;
			
		}
		
        for (int i = 0;i <n;i++)
        {
            cout << arr[i];
        }

	}
}